scripts/aws/aws_helper.py

```diff
@@ -85,7 +85,7 @@ def parse_path(path: str) -> S3Path:
         path (str): A S3 path.
 
     Returns:
-        S3Path (NamedTupe): s3_path.bucket (str), s3_path.key (str)
```

Comment:
not 100% related to this, but does vscode not detect automatically the types of all of these, we shouldnt be doubling up the typing in both the function definition and the docs as it often leads to them being out of sync.